draft-ietf-webpush-protocol.xml

```diff
@@ -691,9 +691,10 @@ Topic = token / quoted-string
           A push message replacement request creates a new push message resource
           and simultaneously deletes any existing message resource that has a
           matching topic.  In effect, the information that is stored for the
-          push message is replaced, but a new resource is created to avoid
-          problems with in flight acknowledgments for the old message.  The push
-          service MAY suppress delivery receipts for the replaced message.
+          push message - including TTL, Urgency and any receipt subscription -

```

Comment:
How about:

A push message replacement request creates a new push message resource and simultaneously deletes any existing message resource has a matching topic. Delivery receipts for the deleted message SHOULD be suppressed.

The replacement request also replaces the stored TTL, Urgency, and any receipt subscription associated with the previous message in the matching topic.